code/modules/mob/living/living_movement.dm

```diff
@@ -58,3 +58,7 @@
 	if(stat > SOFT_CRIT)
 		return
 	return ..()
+
+/// Used as a callback proc by handle_forced_open_adjacency under firedoor.dm
```

Comment:
This is a bad doc comment. You're describing an implementation detail that almost certainly won't be changed when someone touches your code.

Document what the proc *actually* is--a way to check if the user is standing up.